subiquity/common/types/storage.py

```diff
@@ -236,6 +236,8 @@ class GuidedResizeValues:
 @attr.s(auto_attribs=True)
 class GuidedStorageTargetReformat:
     disk_id: str
+    # ptable=None means to use the default (GPT in most scenarios)
+    ptable: Optional[str] = None
```

Comment:
You're absolutely correct that this exposes the capability to autoinstall users of layout.  The following gave me a msdos ptable as expected:

```
autoinstall:
  storage:
    layout:
      ptable: msdos
      name: direct
```
While I'll admit to mixed feelings about that, this will be interesting to some users so we should document that.  Would you add a note about that in the autoinstall references?

Reply:
updated